### PR TITLE
Webots simulation fixes

### PR DESCRIPTION
**Pull request by filiparag**
image

- [x] Fix Exhibition Hall
- [x] Statuette Support PROTO
- [x] Fix Statuette Support boundingObject
- [x] Outer Dispenser PROTO
- [x] Fix Outer Dispenser colors
- [x] Set better Viewpoint
- [x] Start wall markers

### CODE REVIEW

**Comment by VladimirVincan** [comment]
Potencijalno srediti PCB?

- [ ] Add better image
- [ ] Add PCB bounding object
- [ ] Add resistor bounding object
- [ ] Add touch sensors

**Comment by filiparag** [comment]
> Potencijalno srediti PCB?

Ostavio bih ovo za PR u kom dodajemo merenje otpornosti

File: mep3_simulation/webots_data/worlds/eurobot_2022.wbt [hunk 1]
```diff
@@ -12,8 +12,8 @@ WorldInfo {
   ]
 }
 Viewpoint {
-  orientation -0.41854013086365666 0.5616771954331167 0.7136826234307667 1.6139520855172702
-  position -0.7622138942763427 -1.5064143216148775 4.558290954472648
+  orientation -0.33 0.33 1 1.67
+  position 0 -4.2 3.5
```
Comment (VladimirVincan) [inline mep3_simulation/webots_data/worlds/eurobot_2022.wbt:16]: Sliku sa novim viewpointom staviti u README.md https://github.com/memristor/mep3/blob/c0d4ccabc69f98dc081f8a1200dba741addcc9fa/README.md?plain=1#L3

**Comment by VladimirVincan** [comment]
Koliko vidim, još jedino nedostaju trake iznad početnih pozicija:
image
image

**Comment by VladimirVincan** [comment]
Sve je odlično urađeno. Jedino bih OpponentBox-ove i naše robote stavio da se nalaze na samom kraju wbt fajla, da ne moramo da ih tražimo ako ćemo nešto da izmenimo tipa kontroler, ali to može u robot_update PR-u.

**Review by VladimirVincan — APPROVED** [review]

**Comment by filiparag** [comment]
- Fix Exhibition Hall
- Statuette Support PROTO
- Fix Statuette Support boundingObject
- Outer Dispenser PROTO
- Fix Outer Dispenser colors
- Set better Viewpoint
- Start wall markers

**Pull request merged**